compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -1723,3 +1724,62 @@ extern "C" LLVMValueRef
 LLVMRustBuildMaxNum(LLVMBuilderRef B, LLVMValueRef LHS, LLVMValueRef RHS) {
     return wrap(unwrap(B)->CreateMaxNum(unwrap(LHS),unwrap(RHS)));
 }
+
+struct LLVMRustCOFFShortExport {
+    const char* name;
```

Comment:
FTR most of this file uses two space indent. The code immediately above is an exception.